src/librustc_errors/lib.rs

```diff
@@ -705,6 +747,7 @@ impl Handler {
     }
 
     pub fn abort_if_errors(&self) {
+        self.emit_stashed_diagnostics();
```

Comment:
Maybe not necessary and covered by `FatalError.raise()` below which should result in `.print_error_count` doing this, but better safe then sorry?